merlin/systems/dag/ops/operator.py

```diff
@@ -62,7 +62,10 @@ def transform(
         DataFrame
             Returns a transformed dataframe for this operator
         """
-        raise NotImplementedError
+        raise NotImplementedError(
+            f"{self.__class__.__name__} does not have a Transform function."
+            "Please create one, if required."
```

Comment:
I think I might be confused about what this is telling me to do if I saw this error message. It seems like one of the important details to include might be about base operators vs runtime operators? Like...where should I add a `transform` method if I see this error?